components/reducers/contentManagerReducer.js

```diff
@@ -0,0 +1,73 @@
+export const initialState = {
+  values: [],
+  columns: [],
+  rows: [],
+  entity_type_name: null,
+  isLoading: false,
+  isRefresh: true,
+};
+
+const LOADING = "LOADING";
```

Comment:
Kindly refactor this to a separate file. You can add it to reducers > actions.js